<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>licensePlateDialog</class>
 <widget class="QDialog" name="licensePlateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>licensePlateDialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>771</width>
     <height>581</height>
    </rect>
   </property>
   <widget class="QWidget" name="cameraPage">
    <widget class="QScrollArea" name="m_camArea">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>551</width>
       <height>291</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>549</width>
        <height>289</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>410</y>
       <width>351</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="m_btnStart">
        <property name="text">
         <string>Camera Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_btnStop">
        <property name="text">
         <string>Camera Stop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_btnCapture">
        <property name="text">
         <string>Capture</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="payPage">
    <widget class="QPushButton" name="m_btn0">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>160</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_btn1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>160</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_btn4">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>160</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_btn2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>160</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_btn3">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>160</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_btn5">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>160</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_btn8">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>160</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_btn7">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>160</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_btn6">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>160</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_btn9">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>220</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_btnA">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>220</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>A</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_btnB">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>220</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>B</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_btnC">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>220</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>C</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_btnD">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>220</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>D</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_btnE">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>220</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>E</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_btnF">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>220</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>F</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_btnG">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>220</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>G</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_btnH">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>220</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>H</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_btnI">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>280</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>I</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_btnJ">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>280</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>J</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_btnK">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>280</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>K</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_btnL">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>280</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>L</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_btnM">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>280</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>M</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_btnN">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>280</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>N</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_btnO">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>280</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>O</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_btnP">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>280</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>P</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_btnQ">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>280</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Q</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_btnR">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>340</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>R</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_btnS">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>340</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>S</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_btnT">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>340</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>T</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_btnU">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>340</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>U</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_btnV">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>340</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>V</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_btnW">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>340</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>W</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_btnBack">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>470</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;-</string>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="m_btnClear">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>470</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>clear</string>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
    <widget class="QTextEdit" name="m_txtResult">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>391</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="m_btnOK">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="m_btnX">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>340</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_btnY">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>340</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m_btnZ">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>340</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
